engines/mus.cpp

```diff
@@ -68,6 +68,11 @@ namespace pono {
     return makeControlVar(constraintTypeToStr[type] + "_" + s);
   }
 
+  Term Mus::makeControlVar(ConstraintType type, const string s)
```

Comment:
- Consider calling this function from `Mus::makeControlVar` on line 62 as `return makeControlVar(type, s)`
- Consider renaming `s` to something more descriptive like 'suffix'

Reply:
I wonder if having a different function called `string Mus::controlRepr(...)` makes sense, which returns the string identifier for these various types.

Then, you can have a single function that does `return makeControlVar(controlRepr(...))` and the `controlRepr` functions can compose nicely via string operations.

It may not be worth the effort here, but that's how I would organize this pattern if I expected many more cases like this.

Reply:
> * Consider calling this function from `Mus::makeControlVar` on line 62 by replacing line 68 with `return makeControlVar(type, s)`
> 
>     * Consider renaming `s` to something more descriptive like 'suffix'

fixed these in 27be8d4, 775436f